Apps/RestServer/src/main/java/rapture/server/rest/RestServer.java

```diff
@@ -236,9 +239,125 @@ private void setupRoutes() {
             }
             return Kernel.getDecision().createWorkOrder(getContext(req), getWorkorderUriParam(req), params);
         });
-
+        
+        post("/sstore/:authority", (req, res) -> {
+        	log.info(req.body());
+            Map<String, Object> data = JacksonUtil.getMapFromJson(req.body());
+            String authority = req.params(":authority");
+            String config = (String) data.get("config");
+            CallingContext ctx = getContext(req);
+            if (Kernel.getStructured().structuredRepoExists(ctx, authority)) {
+                halt(409, String.format("Repo [%s] already exists", authority));
+            }
+            Kernel.getStructured().createStructuredRepo(ctx, authority, config);
+            return new RaptureURI(authority, Scheme.STRUCTURED).toString();
+        });
+        
+        post("/sstore/:authority/:table", (req, res) -> {
+        	log.info(req.body());
+        	Map<String, Object> data = JacksonUtil.getMapFromJson(req.body());
```

Comment:
I was missing the incapture style xml. so loaded that and then update the save actions setting.